common/src/stack/command/stack/commands/load/json/plugin_network.py

```diff
@@ -0,0 +1,96 @@
+# @copyright@
+# Copyright (c) 2006 - 2018 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
+
+import stack.commands
+import json
+from stack.exception import CommandError
+
+class Plugin(stack.commands.Plugin, stack.commands.Command):
+	notifications = True
+
+	def provides(self):
+		return 'network'
+
+	def requires(self):
+		return [ 'software', 'environment', 'group' ]
+
+	def run(self, args):
+
+		# check if the user would like to import software data
+		# if there are no args, assume the user would like to import everthing
+		if args and 'network' not in args:
+			return
+
+		# self.owner.data contains the data from the json file defined in init
+		if 'network' in self.owner.data:
+			import_data = self.owner.data['network']
+		else:
+			self.owner.log.info('no network data in the json document')
+			return
+
+		self.notify('\n\tLoading network\n')
+		# TODO: sanitize validate
+		for network in import_data:
+			name = network['name'].strip()
+			try:
+				# the add network command requires at least name address and mask
+				# if the network exists already we want to overwrite its information
+				# so we first add the network then set everything else
+				self.owner.command('add.network', [
+								name,
+								f'address={network["address"]}',
+								f'mask={network["netmask"]}'
+				])
+				self.owner.log.info(f'success adding network {name}')
+				self.owner.successes += 1
+			except CommandError as e:
+				if 'exists' in str(e):
+					self.owner.log.info(f'warning adding network {name}: {e}')
+					self.owner.warnings += 1
+				else:
+					self.owner.log.info(f'error adding network {name}: {e}')
+					self.owner.errors += 1
+
+			try:
+				# now we set all of the attributes
+				# if the network existed already in the database we will overwrite everything about it
```

Comment:
Does this work if the network exists already and the input data being loaded is empty?  For example, `public` exists with `dns=True`, but the json lacks that field?  This is a good integration test.